overdrive.py

```diff
@@ -118,7 +118,6 @@ def environment_values(cls):
     def from_environment(cls, _db):
         # Make sure all environment values are present. If any are missing,
         # return None. Otherwise return an OverdriveAPI object.
-        values = cls.environment_values()
```

Comment:
It's not actually used elsewhere in the `from_environment` method. It's called in the `__init__` method instead, when it's invoked on line 123.